### PR TITLE
Add detail for parameter s3ForcePathStyle in MinIO page.

### PR DESCRIPTION
**Pull request by blackpiglet**
Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [x] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Review by ywk253100 — DISMISSED** [review]

**Review by allenxu404 — APPROVED** [review]

File: site/content/docs/main/contributions/minio.md [hunk 1]
```diff
@@ -81,11 +81,16 @@ These instructions start the Velero server and a Minio instance that is accessib
...
 
-    Additionally, you can specify `--use-node-agent` to enable File System Backup support, and `--wait` to wait for the deployment to be ready.
+    * Additionally, you can specify `--use-node-agent` to enable File System Backup support, and `--wait` to wait for the deployment to be ready.
 
-    This example also assumes you have named your Minio bucket "velero".
+    * This example also assumes you have named your Minio bucket "velero".
+
+    * Please make sure to set parameter `s3ForcePathStyle=true`. The parameter is used to set the Velero integrated AWS SDK data query address style. There are two types of the address: [virtual-host and path-style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html). If the `s3ForcePathStyle=true` is not set, the default value is false, then the AWS SDK will query in virtual-host style, but the MinIO server only support path-style address by default. The miss match will cause Velero can upload data to MinIO, but cannot download from MinIO. This [link](https://github.com/vmware-tanzu/velero/issues/7268) is an example of this issue.
```
Comment (kaovilai) [inline site/content/docs/main/contributions/minio.md:90]:
```suggestion
    * Please make sure to set parameter `s3ForcePathStyle=true`. The parameter is used to set the Velero integrated AWS SDK data query address style. There are two types of the address: [virtual-host and path-style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html). If the `s3ForcePathStyle=true` is not set, the default value is false, then the AWS SDK will query in virtual-host style, but the MinIO server only support path-style address by default. The miss match will mean Velero can upload data to MinIO, but **cannot download from MinIO**. This [link](https://github.com/vmware-tanzu/velero/issues/7268) is an example of this issue.
```

File: site/content/docs/v1.13/contributions/minio.md [hunk 2]
```diff
@@ -81,11 +81,16 @@ These instructions start the Velero server and a Minio instance that is accessib
...
 
-    Additionally, you can specify `--use-node-agent` to enable File System Backup support, and `--wait` to wait for the deployment to be ready.
+    * Additionally, you can specify `--use-node-agent` to enable File System Backup support, and `--wait` to wait for the deployment to be ready.
 
-    This example also assumes you have named your Minio bucket "velero".
+    * This example also assumes you have named your Minio bucket "velero".
+
+    * Please make sure to set parameter `s3ForcePathStyle=true`. The parameter is used to set the Velero integrated AWS SDK data query address style. There are two types of the address: [virtual-host and path-style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html). If the `s3ForcePathStyle=true` is not set, the default value is false, then the AWS SDK will query in virtual-host style, but the MinIO server only support path-style address by default. The miss match will cause Velero can upload data to MinIO, but cannot download from MinIO. This [link](https://github.com/vmware-tanzu/velero/issues/7268) is an example of this issue.
```
Comment (kaovilai) [inline site/content/docs/v1.13/contributions/minio.md:90]:
```suggestion
    * Please make sure to set parameter `s3ForcePathStyle=true`. The parameter is used to set the Velero integrated AWS SDK data query address style. There are two types of the address: [virtual-host and path-style](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html). If the `s3ForcePathStyle=true` is not set, the default value is false, then the AWS SDK will query in virtual-host style, but the MinIO server only support path-style address by default. The miss match will mean Velero can upload data to MinIO, but **cannot download from MinIO**. This [link](https://github.com/vmware-tanzu/velero/issues/7268) is an example of this issue.
```

**Review by kaovilai — CHANGES_REQUESTED** [review]

**Comment by mateusoliveira43** [comment]
Is this PR needed? Is not the docs built from main branch, and in there, documentation for 1.13 is up to date?

**Comment by blackpiglet** [comment]
> Is this PR needed? Is not the docs built from main branch, and in there, documentation for 1.13 is up to date?

That's true. It's not required.

File: site/content/docs/main/contributions/minio.md (hunk 1 above)
Comment (blackpiglet) [reply site/content/docs/main/contributions/minio.md]: Modified.

**Review by kaovilai — APPROVED** [review]

**Review by ywk253100 — APPROVED** [review]

**Review by qiuming-best — APPROVED** [review]

**Pull request merged**